src/event-schemas/cumulative-layout-shift-event.json

```diff
@@ -11,7 +11,36 @@
         },
         "value": {
             "type": "number",
-            "description": "Value of the cls metric"
+            "description": "Largest burst of unexpected layout shifts during a page's lifespan"
+        },
+        "attribution": {
+            "type": "object",
+            "description": "Attributions for CLS",
+            "properties": {
+                "largestShiftTarget": {
+                    "type": "string",
+                    "description": "First element in the largest layout shift contributing to CLS score"
+                },
+                "largestShiftValue": {
+                    "type": "number",
+                    "description": "Value of CLS' single largest shift"
+                },
+                "largestShiftTime": {
```

Comment:
We may want to consider removing shiftTime because loadState makes this timestamp somewhat redundant